forwards_paper.tex

```diff
@@ -86,53 +77,47 @@ \section*{Introduction}
 However, these powerful advances came with substantial caveats:
 the backwards-in-time processes that are described by coalescent theory
 are only \emph{Markovian}, and thus feasible to work with,
-thanks to the important assumptions of (a) random mating, and (b) neutrality.
-\plr{Brief statement why this is.  Also include stationarity?}
-Both assumptions can be side-stepped to a limited extent, and so coalescent methods are now commonly used to
-simulate the results of population dynamics of a collection of randomly mating populations exchanging migrants,
-having a small number of loci under selection.
-Mapping results of such models onto real species can be challenging,
-as these are often distributed across geographical space and may have large numbers of loci under various sorts of selection.
+thanks to the important assumptions of 
+(a) random mating, 
+(b) neutrality,
+and (c) small sample size relative to the population size.
+The first two assumptions can be side-stepped to a limited extent, 
+but it remains a challenge to map results of coalescent models
+onto species that are distributed across continuous geographical space 
+and/or have large numbers of loci under various sorts of selection.
 Furthermore, the relationship between the life history of a species --
 fecundity and mortality schedules, allee effects, and demographic fluctuations --
 are all absorbed into a single compound parameter, the coalescence rate.
+The third assumption is no longer safe, either --
+for example, a recent study~\citep{martin2017human}
+simulated 600,000 samples of human chromosome 20
+% to assess the impact of European biased reference panels in genome wide association studies. 
+Several studies have now shown that in samples of size approaching that of the population,
+genealogical properties may be distorted relative to the coalescent expectation
+\citep{wakeley2003gene,maruvka2011recovering,bhaskar2014distortion}.
 These considerations, and increasing computational power, have led to a resurgence of interest in forwards-time, individual-based simulations.
 
-Another critical assumption of the coalescent is that the sample size $n$ is much smaller
```

Comment:
I merged this into the previous paragraph (editing for brevity; but we could put more details back in); and moved the "hybrid approach" to the discussion.